src/main/java/org/sonarlint/intellij/finding/tracking/Tracker.java

```diff
@@ -373,7 +373,7 @@ private static class ServerIssueSearchKey implements SearchKey {
     private final String serverIssueKey;
 
     ServerIssueSearchKey(Trackable trackable) {
-      serverIssueKey = trackable.getServerIssueKey();
+      serverIssueKey = trackable.getServerFindingKey();
```

Comment:
I don't think `serverIssueKey` makes sense if we're using Finding now, what about renaming inner class `ServerIssueSearchKey`  and enum `ServerIssueSearchKeyFactory`

Reply:
Yes